Neos.ContentRepository/Classes/Security/Authorization/Privilege/Node/EditNodePrivilege.php

```diff
@@ -32,7 +32,7 @@ public function matchesSubject(PrivilegeSubjectInterface $subject)
             throw new InvalidPrivilegeTypeException(sprintf('Privileges of type "%s" only support subjects of type "%s" or "%s", but we got a subject of type: "%s".', EditNodePrivilege::class, NodePrivilegeSubject::class, MethodPrivilegeSubject::class, get_class($subject)), 1417017239);
         }
 
-        $this->initialize();
+        $this->initializeMethodPrivilege();
```

Comment:
this could go into the condition like in RemoveNodePrivilege right?

Reply:
Sure, can do.